pkg/synopsysctl/cmd_db_migrate.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright (C) 2019 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package synopsysctl
+
+import (
+	"fmt"
+
+	"github.com/blackducksoftware/synopsys-operator/pkg/util"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// dbMigrateCmd puts a resource into the db-migrate state
+var dbMigrateCmd = &cobra.Command{
+	Use:     "db-migrate",
+	Aliases: []string{"database-migrate"},
+	Short:   "Put a Synopsys resource into the database-migrate state",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return fmt.Errorf("must specify a resource")
+	},
+}
+
+// dbMigrateBlackDuckCmd puts a Black Duck instance into the cluster
+var dbMigrateBlackDuckCmd = &cobra.Command{
+	Use:     "blackduck NAMESPACE",
+	Example: "synopsysctl db-migrate blackduck <namespace>",
+	Short:   "Put a Black Duck instance into the database-migrate state",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
```

Comment:
Why not allow 1 or more?

Reply:
@rrati stop and start cannot currently do multiple. For this PR should I make them all do multiple, only make migrate do multiple, or make an issue to make all these commands do multiple in a new PR?

Reply:
Create a separate PR to make start/stop/db-migrate take more than 1 target